wherehows-web/app/components/datasets/containers/dataset-health.ts

```diff
@@ -163,4 +179,11 @@ export default class DatasetHealthContainer extends Component {
       currentSeverityFilter: filterType === 'severity' && newFilterName !== currentSeverityFilter ? newFilterName : ''
     });
   }
+
+  constructor() {
+    super(...arguments);
+    console.log('constructing');
```

Comment:
oops `console.log`, i'll add a lint rule for this

Reply:
Should have done a better job of tracking these down woops